src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { TodowithUser } from '../types';
+
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
please, prefer the interface